development/migrations/dependencies.rst

```diff
@@ -0,0 +1,100 @@
+
+In your Migration file, add a static public function named depends_on(), returning an array.
+
+.. code-block:: php
+
+    static public function depends_on()
+    {
+        return array();
+    }
+
+All you must do is add the class name of the Migration that the current Migration depends on.
+
+For example, ``phpbb\db\migration\data\v310\dev`` that 3.0.11, extensions, style update,
+reported posts display, and timezone are installed.
+
+.. code-block:: php
+
+    static public function depends_on()
+    {
+        return array(
+            '\phpbb\db\migration\data\v310\extensions',
+            '\phpbb\db\migration\data\v310\style_update_p2',
+            '\phpbb\db\migration\data\v310\timezone_p2',
+            '\phpbb\db\migration\data\v310\reported_posts_display',
+            '\phpbb\db\migration\data\v310\migrations_table',
+        );
+    }
+
+**Note:** It is highly recommended to add a dependency for all migrations. If the migration
+is the first migration of your extension just depend on the release migration of the minimum
+phpBB requirement of your extension, e.g. 3.1.4:
+
+.. code-block:: php
+
+    static public function depends_on()
+    {
+        return array(
+            '\phpbb\db\migration\data\v31x\v314',
+        );
+    }
+
+depends_on creates a tree
+=========================
+
+Imagine the following situation:
+
++-------------+---------------------------------------+----------------------------------------------------+
+| Migration   | Dependencies                          | Note                                               |
++=============+=======================================+====================================================+
+| migration_1 | phpbb\\db\\migration\\data\\v310\\dev | Initial release, 1.0.0, requires phpBB 3.1.0       |
++-------------+---------------------------------------+----------------------------------------------------+
+| migration_2 | migration_1                           | New feature for 1.0.1 (during development)         |
++-------------+---------------------------------------+----------------------------------------------------+
+| migration_3 | migration_1                           | Another new feature for 1.0.1 (during development) |
++-------------+---------------------------------------+----------------------------------------------------+
+| migration_4 | migration_2                           | Modifying the feature in migration_2               |
++-------------+---------------------------------------+----------------------------------------------------+
+| migration_5 | migration_3, migration_4              | Release 1.0.1                                      |
++-------------+---------------------------------------+----------------------------------------------------+
+
+This dependency setup would cause the following Migrations to be installed when an individual
+Migration was installed. *Assumes* ``phpbb\db\migration\data\v310\dev`` *is already installed.*
```

Comment:
"This dependency setup would cause the following Migrations to be installed when an individual Migration was installed."

The end of this sentence is confusing.